plugins/plugin-core-event-listener-graphing/package.json

```diff
@@ -49,10 +49,10 @@
     "@syntest/analysis": "^0.1.0-beta.4",
     "@syntest/base-language": "*",
     "@syntest/cfg": "*",
+    "@syntest/search": "*",
```

Comment:
For some reason the search keeps getting placed out of order

Reply:
Fixed.